miqa/core/tests/test_rest.py

```diff
@@ -62,6 +62,34 @@ def test_projects_list(user_api_client, project, user):
         }
 
 
+@pytest.mark.django_db
+def test_project_status(
+    user_api_client,
+    project,
+    experiment,
+    scan_factory,
+    scan_decision_factory,
+    user_factory,
+):
+    # 3 of 5 scans marked as complete
+    decisions = [
+        ('U', 'tier_1_reviewer'),
+        ('U', 'tier_2_reviewer'),
+        ('UN', 'tier_1_reviewer'),  # needs tier 2 review
+        ('UN', 'tier_2_reviewer'),
+        ('Q?', 'tier_1_reviewer'),  # needs tier 2 review
+    ]
+    scans = [scan_factory(experiment=experiment) for i in range(len(decisions))]
+    for i, scan in enumerate(scans):
+        decider = user_factory()
+        assign_perm(decisions[i][1], decider, project)
+        scan_decision_factory(scan=scan, creator=decider, decision=decisions[i][0])
+    resp = user_api_client().get(f'/api/v1/projects/{project.id}')
+    if resp.status_code == 200:
+        assert resp.data['status']['total_scans'] == len(scans)
+        assert resp.data['status']['total_complete'] == 3
```

Comment:
Rather than testing with the API client, let's test the `project.get_status()` method directly, since the value of this test is in validating the query logic.

Reply:
Changed in [491a19d](https://github.com/OpenImaging/miqa/pull/567/commits/491a19da608cc75902e53ae9f80bf0327f45144b)